src/lightning_app/utilities/app_logs.py

```diff
@@ -40,6 +40,41 @@ def __gt__(self, other: "_LogEvent") -> bool:
         return self.timestamp > other.timestamp
 
 
+class LogSection:
+    """Delimiters used to separate sections of Lightning App logs."""
+
+    USER_RUN_FLOW = "USER_RUN_FLOW"
```

Comment:
We should probably show the user dependency installation as well

Reply:
@hhsecond Getting the CLI to have complete parity with how the UI displays logs is a bigger task than this pull request was intended to cover.

Reply:
@tchaton

Reply:
Hey, @alecmerdler We should do this on the platform side. It doesn't make sense to filter the logs with the private indications in an open-source project.

Reply:
@tchaton This logic is just copied from what we do in the UI, which any dedicated person could reverse-engineer. It's significantly more work to completely filter internal things from the external logs API and also have our nice app startup progress indicator. This was intended to be a :bicyclist: solution that would last us a few months until we had time to implement the next iteration.